mscore/plugin/api/elements.h

```diff
@@ -114,6 +115,11 @@ class Element : public Ms::PluginAPI::ScoreElement {
        * \since MuseScore 3.3
        */
       Q_PROPERTY(qreal posY READ posY)
+      /**
+       * Position of this element in page coordinates, in spatium units.
```

Comment:
`(read only)`, right?

Reply:
Actually some "read-only" marker will also be generated by Doxygen so there is no strict necessity in writing that explicitly. Other similar properties in `Element` class do not have `read-only` in their description. Maybe it would be better to remove that from `pagenumber` description then.

Reply:
OK, either way works for me ;-)